content/en/quickstart/_index.md

```diff
@@ -83,3 +81,9 @@ The error codes of the found issues are prefixed by 'IST' or 'KIA'. For the deta
 - https://istio.io/latest/docs/reference/config/analysis/ for 'IST' error codes
 - https://kiali.io/documentation/latest/validations/ for 'KIA' error codes
 ```
+
+<h3>Next Steps</h3>
```

Comment:
since this is markdown, why no use "###" instead of "<h3>"?

Reply:
see comment above, will correct